components/new-contribution-flow/ContributionFlowSuccess.js

```diff
@@ -225,19 +229,19 @@ class NewContributionFlowSuccess extends React.Component {
     if (!LoggedInUser) {
       return null;
     }
-    return (
-      <PublicMessageForm
-        stepProfile={this.props.stepProfile}
-        publicMessage={publicMessage}
-        collective={this.props.collective}
-      />
-    );
+    return <PublicMessageForm order={this.props.data.order} publicMessage={publicMessage} />;
   };
 
   render() {
-    const { loadingLoggedInUser, LoggedInUser, data, collective, contribution } = this.props;
+    const { loadingLoggedInUser, LoggedInUser, collective, data } = this.props;
+    const { order } = data;
     const shareURL = `${process.env.WEBSITE_URL}${collective.path}`;
-    const publicMessage = data && get(data, 'member.publicMessage', null);
+
+    const getPublicMessage = order => {
+      const memberData = first(get(order, 'fromAccount.memberOf.nodes', null));
+      const publicMessage = memberData && get(memberData, 'publicMessage', null);
+      return publicMessage;
```

Comment:
`get` is safe to use against missing data, you don't need the `memberData &&`. It's also powerful enough for you to write directly:

```suggestion
      return get(order, 'fromAccount.memberOf.nodes[0].publicMessage')
```

Reply:
Ok that's good to know! I wasn't sure about using `nodes[0]` inline and thought using `first` was preferred but I'll be sure to make full use of `get` in the future 🎉